integration/internal/exec/exec.go

```diff
@@ -0,0 +1,85 @@
+package exec
```

Comment:
There are some lint errors because of naming.

I think we can put this into `integration/internal/container` so that calling it `Exec` is fine.

Reply:
Sure, will do